File: move.py
import os, shutil

os.makedirs('api/_youtube', exist_ok=True)

for f in os.listdir('api'):
    if f.startswith('youtube-') and f.endswith('.ts'):
        shutil.move(f'api/{f}', f'api/_youtube/{f}')
        print(f"Moved {f}")

for f in ['blobs.ts', 'migrate-blobs-to-redis.ts']:
    path = f'api/{f}'
    if os.path.exists(path):
        os.remove(path)
        print(f"Deleted {f}")

print("Python script completed.")

File: move.js
const fs = require('fs');
const path = require('path');

const apiDir = path.join(__dirname, 'api');
const targetDir = path.join(apiDir, '_youtube');

if (!fs.existsSync(targetDir)) {
    fs.mkdirSync(targetDir);
}

const files = fs.readdirSync(apiDir);
files.forEach(file => {
    if (file.startsWith('youtube-') && file.endsWith('.ts')) {
        const oldPath = path.join(apiDir, file);
        const newPath = path.join(targetDir, file);
        fs.renameSync(oldPath, newPath);
        console.log(`Moved ${file} to _youtube/`);
    }
});

const toDelete = ['blobs.ts', 'migrate-blobs-to-redis.ts'];
toDelete.forEach(file => {
    const fPath = path.join(apiDir, file);
    if (fs.existsSync(fPath)) {
        fs.unlinkSync(fPath);
        console.log(`Deleted ${file}`);
    }
});
